src/main/include/log4cxx/rolling/multiprocessrollingfileappender.h

```diff
@@ -34,12 +34,12 @@ namespace rolling
 /**
  * A special version of the RollingFileAppender that acts properly with multiple processes
  */
-class LOG4CXX_EXPORT MultiprocessRollingFileAppender : public FileAppender
```

Comment:
I'm pretty sure this breaks the ABI(changing the class hierarchy), but the MultiprocessrollingFileAppender has never been tracked as part of the ABI because it is optional.  Are we concerned about any ABI breakage or do we not care because it's optional?